src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius/sylius_mailer.yml

```diff
@@ -25,9 +25,13 @@ sylius_mailer:
         password_reset:
             subject: sylius.emails.user.password_reset.subject
             template: "@SyliusCore/Email/passwordReset.html.twig"
+        #deprecated since Sylius 1.13 and will be removed in 2.0, use account_verification instead
         account_verification_token:
```

Comment:
Maybe let's add an information about it to upgrade file? Putting just a comment here doesn't look like an effective way to inform people about it.